tests/test_gshcablobstore.py

```diff
@@ -17,6 +19,7 @@
 
 class TestGSHCABlobStore(unittest.TestCase, HCABlobStoreTests):
     def setUp(self):
```

Comment:
I'm a little concerned that these tests could become flaky. Let's upload `Deletion_test.txt` here to make sure it always exists.